modules/Microsoft.Sql/servers/encryptionProtector/deploy.bicep

```diff
@@ -0,0 +1,53 @@
+@description('Conditional. The name of the sql server.')
```

Comment:
Hey @jhueppauff please also add a `version.json` file

Reply:
I hope I created it the correct way